salesforce/10.22/modules/ROOT/pages/salesforce-connector-processing-events.adoc

```diff
@@ -115,7 +115,7 @@ If there is a connection issue or the token expires and the connector must resub
 ====
 The connector interacts with Object Store to store and retrieve data only when the *Replay Option* field is set to `FROM_LAST_REPLAY_ID` or when the *Replay failed events if any or resume from last replay id* checkbox is selected.
 
-In CloudHub, when the *Replay Option* field is set to `FROM_LAST_REPLAY_ID` or when the *Replay failed events if any or resume from last replay id* checkbox is selected, the last known replay ID might not be found in Object Store, which causes all events to replay when the application restarts or redeploys. This is because Object Store v2 does not have a distributed locking mechanism. For more information, refer to https://help.mulesoft.com/s/article/Salesforce-Connector-Last-known-replayId-not-found-in-ObjectStore[Last known replayId not found in ObjectStore^].
+In CloudHub 1.0 and CloudHub 2.0, when the *Replay Option* field is set to `FROM_LAST_REPLAY_ID` or when the *Replay failed events if any or resume from last replay id* checkbox is selected, the last known replay ID might not be found in Object Store, which causes all events to replay when the application restarts or redeploys. This is because Object Store v2 does not have a distributed locking mechanism. For more information, refer to https://help.mulesoft.com/s/article/Salesforce-Connector-Last-known-replayId-not-found-in-ObjectStore[Last known replayId not found in ObjectStore^].
```

Comment:
```suggestion
In CloudHub and CloudHub 2.0, when the *Replay Option* field is set to `FROM_LAST_REPLAY_ID` or when the *Replay failed events if any or resume from last replay id* checkbox is selected, the last known replay ID might not be found in Object Store, which causes all events to replay when the application restarts or redeploys. This is because Object Store v2 does not have a distributed locking mechanism. For more information, refer to https://help.mulesoft.com/s/article/Salesforce-Connector-Last-known-replayId-not-found-in-ObjectStore[Last known replayId not found in ObjectStore^].
```